core/src/main/java/hudson/model/Queue.java

```diff
@@ -2672,6 +2707,20 @@ public void run() {
         }
     }
 
+    private class BuildableRunnable implements Runnable  {
```

Comment:
I don't think the Runnable's are persisted to disk... rather it is the Items... but I'd need to look at the code to confirm by 95% certain as 100% certain